src/main/java/io/pinecone/PineconeIndexOperationClient.java

```diff
@@ -96,9 +107,11 @@ private Request buildRequest(String method, String path, String acceptHeader, Re
         if (HTTP_METHOD_POST.equals(method)) {
             builder.post(requestBody);
             builder.addHeader(CONTENT_TYPE, CONTENT_TYPE_JSON);
-        } else if (HTTP_METHOD_DELETE.equals(method)) {
+        } else if(HTTP_METHOD_PATCH.equals(method)) {
+            builder.patch(requestBody);
+            builder.addHeader(CONTENT_TYPE, CONTENT_TYPE_JSON);
+        } else if (HTTP_METHOD_DELETE.equals(method))
             builder.delete();
-        }
```

Comment:
I actually skipped the brackets completely by mistake which is fine for java since there's only one line after the else-if but definitely not consistent, so let me add it back